python/tk_desktop/desktop_engine_project_implementation.py

```diff
@@ -328,7 +328,7 @@ def _get_setting(self, setting_name, default_value=None):
         """
         return self._engine.get_setting(setting_name, default_value)
 
-    def _emit_log_message(self, level, msg, *args):
+    def _emit_log_message(self, handle, record):
```

Comment:
Don't know what I was thinking when I wrote this originally. It made no sense. This is NOT the signature of `sgtk.platform.Engine._emit_log_message`